code/modules/admin/verbs/onlyone.dm

```diff
@@ -3,12 +3,29 @@
 		alert("The game hasn't started yet!")
 		return
 
+	for(var/mob/living/silicon/S in player_list) //All silicons get made into humans so they can be highlanders, too.
+		if(S.stat == 2 || !(S.client)) continue
```

Comment:
* You shouldn't check `stat` directly, use `isDead()'
* The parenthesis shouldn't be needed.